tests/unit/test_masks.py

```diff
@@ -148,11 +159,39 @@ def it_knows_its_mask_tile(self, request):
         tissue_mask_tile_.return_value = expected_mask
 
         tissue_mask = TissueMask()
-
         binary_mask = tissue_mask(tile)
 
         assert binary_mask == expected_mask
 
+    def it_knows_its_mask_tile_and_compute_with_default(self, request):
+        tile = Tile(PILIMG.RGBA_COLOR_500X500_155_249_240, None, None)
+
```

Comment:
i would remove this line. The reason is that in a test, for the sake of readability and for future developers we're trying to keep separated the 3 blocks that compose a test.
1. Objects declarations
2. Actual call
3. Assertions

Hope this can help